howto/get-secrets/README.md

```diff
@@ -35,7 +35,7 @@ import (
 )
 
 func main() {
```

Comment:
It would be good to add in the docs that following code snippets shows fetching secret from default namespace stored in kubernetes secret store

Reply:
done